### PR TITLE
Add homepage link to base.html

### PR DESCRIPTION
**Pull request by cthoyt**
### Description

Motivation: I was surprised that the homepage for events wasn't shown, so I wanted to add them. I realized that the links bar was actually controlled by `base.html` and it would be possible to do some logic over `item.claims` to get and display the value, which was already available.

This PR adds a new conditional in the javascript part of `base.html` to introspect over `item.claims` and add a homepage to the links bar, if available. This affects all pages derived from `base.html`. Here's how it looks in two pages:

#### Author Page

Screen Shot 2022-09-27 at 12 43 46

#### Event Page

Screen Shot 2022-09-27 at 12 43 31

This PR also enables entrypoints so it's possible to run `scholia run` from the command line after pip installation. There was previous discussion about this in #1471 but for some reason there were issues. I'm hoping that now the project has modernized a bit that this is non-controversial.

### Caveats

> Please list anything which has been left out of this PR or which should be considered before this PR is accepted
Check any of the following which apply:

* [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ]  This change requires a documentation update
    * [ ]  I have made corresponding changes to the documentation
* [ ]  This change requires new dependencies (please list)

*If you make changes to the Python code*
  
* []  My code passes the [tox](https://tox.readthedocs.io/en/latest/) check, I can receive warnings about tests, documentation or both

### Testing
Please describe the [tests](https://numpy.org/doc/stable/reference/testing.html) that you ran to verify your changes. Provide instructions, so we can reproduce. Please also list any relevant details for your test configuration.

N/A

### Checklist

* [x] I have [commented](https://peps.python.org/pep-0008/#comments) my code, particularly in hard-to-understand areas
* [x] My changes generate no new warnings
* [x] I have not used code from external sources without attribution
* [x] I have considered [accessibility](https://www.w3.org/standards/webdesign/accessibility) in my implementation 
* [x] There are no remaining debug statements (print, console.log, ...)

### CODE REVIEW

File: scholia/app/templates/base.html [hunk 1]
```diff
@@ -228,6 +228,12 @@
 	     socialMediaLink(detailsList, user, site);
 	 }
 
+	 /* Official homepage */
+	 if ("P856" in item.claims) {
+	     var homepage = item.claims.P856[0].mainsnak.datavalue.value
```
Comment (cthoyt) [inline scholia/app/templates/base.html:233]: would also be feasible to iterate over all homepages here, if desired.

**Comment by fnielsen** [comment]
You more than one issue implemented here. It is better to split them into separate PRs.

**Comment by fnielsen** [comment]
I am looking at http://127.0.0.1:8100/organization/Q24283660 where there is an issue with rank and preferred item. This seems not to be honored. Perhaps the (other) social media links does neither do this?

**Comment by cthoyt** [comment]
> You more than one issue implemented here. It is better to split them into separate PRs.

I split the code change for the vanity CLI into #2113 

> I am looking at http://127.0.0.1:8100/organization/Q24283660 where there is an issue with rank and preferred item. This seems not to be honored. Perhaps the (other) social media links does neither do this?

I'm sorry but I don't understand what you mean. By rank do you mean the order of the links? I see the twitter icon then the website, which is what I'm expecting. https://scholia.toolforge.org/organization/Q24283660 only shows twitter.

**Comment by fnielsen** [comment]
http://127.0.0.1:8100/organization/Q24283660 has currently "http://www.compute.dtu.dk/forskning/Cogsys" as the link. But this link is deprecated, while https://www.compute.dtu.dk/english/Research/Research-sections/Cogsys is the preferred, see https://www.wikidata.org/wiki/Q24283660#P856

**Pull request merged**